Utilities/Maintenance/ApplyScriptToRemotes.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+#==========================================================================
+#
+#   Copyright Insight Software Consortium
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#==========================================================================*/
+
+
+program_name=$0
+
+function usage {
+  echo "Usage: ${program_name} "
+  echo "  param1 path to the script"
+  echo "  param2 name of the feature branch"
+  echo "  param3 commit message"
+  exit 1
+}
+
+# Display usage
+if [ ${#@} != 3 ]; then
+  usage
+fi
+
+# Make sure we are inside the repository
+cd "${BASH_SOURCE%/*}" &&
+
+remote_modules_path='../../Modules/Remote'
+
+# Ask for GitHub username and password once
+echo "Please provide your GitHub (https://github.com)"
+read -p "username: " username
+read -p "password: " -s password
+
+script=$1
```

Comment:
The **full path of the script** to be applied must be provided. Otherwise, we **could assume that only scripts that dwell in `Utilities` will be applied**, since they anyways will be worthwhile maintaining there for the records and for the sake of our philosophy.